src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -874,4 +874,18 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Label_DeprecationReasons" xml:space="preserve">
     <value>Deprecation reason</value>
   </data>
+  <data name="Accessibility_PackageChanges" xml:space="preserve">
+    <value>Package changes</value>
+  </data>
+  <data name="Accesibility_PackageIdentity" xml:space="preserve">
+    <value>{0} version {1}</value>
+    <comment>{1} = package id, {2} = package version</comment>
+  </data>
+  <data name="Preview_PackageUpdate" xml:space="preserve">
+    <value>{0} version {1} to {2} version {3}</value>
+    <comment>{0} = Old package Id, {1} = Old package version, {2} = New package version, {2} = New package version</comment>
+  </data>
+  <data name="Preview_UnknownProject" xml:space="preserve">
+    <value>Unknown Project</value>
```

Comment:
Does "project" need to be capitalized?

Reply:
The [original string](https://github.com/NuGet/NuGet.Client/blob/a746cd6a01dfd22187375ee9c5a0d9b98676020a/src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/PreviewResult.cs#L35) appears capitalized.  That string is for the corner case where PM UI doesn't know the project name in the Preview Window Dialog. A sample text would look like the following:

```
Unknown Project

Installing:

Microsoft.Azure.KeyVault.Core.1.0.0
WindowsAzure.Storage.9.3.3
```

vs.


```
Unknown project

Installing:

Microsoft.Azure.KeyVault.Core.1.0.0
WindowsAzure.Storage.9.3.3
```

Reply:
I will make it lower case.